### ISSUE TITLE
Guild News not updating and missing guild members looted items.

### ISSUE THREAD

**Issue by alanware78** [Haleth/Aurora#183]
I disable Aurora and this problem is fixed. In the guild news the section where guild member loot is listed doesn't work properly. Upon logging in you can see the list, however, after 15 20 minutes, the list depopulates.  After a /reload the list is populated again, but it depopulates again in short amount of time.

I don't have any lua erros that I know of.

**Comment by Gethe** [comment]
What other addons do you have installed? I'm not seeing this issue with just Aurora enabled.

**Comment by alanware78** [comment]
I have quite a few. However, this didn't start happening until I installed Aurora.
Details
Astral Keys
Dominos
KUI
BigWigs
Clique
DataStore
Elephant
Exorsus Raid Tools
Garrison Mission Manager
GTFO
HHTD
HandyNotes
LittleWigs
LoggerHead
Method Dungeon Tools
Minimap Button Frame
Misspelled
MythicPlus Timer
NameplateAuras
NameplateSCT
OmniCC
oRA3
Postal
Prat
Premade Groups Filter
RaiderIO Check
RCLootCouncil
ShadowedUnit Frames
Simulationcraft
TipTac
Vendomatic
WeakAuras
World Quest Tracker
World Quest List
bdCore
bdGrid

That is most of them. I have been running those for a while now. I added Aurora and the issue started. I disable Aurora and it stops. I use ElvUI on different characters and it doesn't happen.

Thanks for the response.
